bentoml/catboost.py

```diff
@@ -1,7 +1,37 @@
-raise NotImplementedError(
-    f"""\
-Support for "{__name__}" is temporarily unavailable as BentoML transition to the new \
-design in version 1.0.0 release. Before this module is officially implemented in \
-BentoML, users may use Custom Runner as a workaround. Learn more at http://docs.bentoml.org
-"""
-)
+import logging
```

Comment:
No need for backward compatibility warning, we don't have catboost support in 1.0. Imports for __all__ is good enough